libs/datasets/sources/cdc_testing_dataset.py

```diff
@@ -58,7 +58,60 @@ class CDCHistoricalTestingDataset(data_source.CanScraperBase):
     @classmethod
     @lru_cache(None)
     def make_dataset(cls) -> MultiRegionDataset:
-        return modify_dataset(super().make_dataset())
+        ds = super().make_dataset()
+        return dataclasses.replace(
+            ds, timeseries=remove_trailing_zeros(ds.timeseries), timeseries_bucketed=None
+        )
+
+
+class CDCOriginallyPostedTestingDataset(CDCHistoricalTestingDataset):
```

Comment:
Not a big deal and maybe this is common in python, but it feels like a bit of a hack to use inheritance like this as a means of sharing code even though there isn't a clear parent class / subclass relationship here.  I think it would be cleaner to create a separate `CDCTestingDatasetBase` base class or something.

Reply:
yeah I agree, done 🙏